src/components/TheNavBar.vue

```diff
@@ -59,10 +61,20 @@ const placeHolder = computed(() => {
       <div class="navbar-center flex">
         <ul class="menu menu-horizontal text-lg">
           <li>
-            <RouterLink :to="{ name: ROUTE_SERVICE_CATALOG }" :props="{ loggedIn }"><a>Services</a></RouterLink>
+            <RouterLink :to="{ name: ROUTE_SERVICE_CATALOG }" :props="{ loggedIn }">
+              Services
+            </RouterLink>
+          </li>
+          <li v-if="loggedIn && userRole == ROLE_ADMIN">
+            <RouterLink :to="{ name: ROUTE_SERVICE_CATALOG }"> <!-- TODO -->
```

Comment:
Why routing to patient management here?

Reply:
Just for now, have added a todo to correct it